packages/data-explorer/src/components/SearchComponent.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div class="input-group mb-3">
+    <input
+      type="text"
+      v-model="searchText"
+      class="form-control"
+      placeholder="Search"
+      aria-label="Search data"
+      aria-describedby="mg-data-explorer-search"
+    />
+    <div class="input-group-append">
+      <button
+        class="btn btn-outline-secondary"
+        type="button"
+        id="mg-data-explorer-search"
+        @click="handleSearchAction"
+      >
+        <font-awesome-icon icon="search"></font-awesome-icon>
+      </button>
+    </div>
+  </div>
+</template>
```

Comment:
![feedback](https://user-images.githubusercontent.com/46395349/76949438-d278ac00-6908-11ea-889a-6377323b0d92.png)

I think that if the header items are a bit better aligned to the block below that the whole thing would look a bit less cluttered and more professional.

Even if we world only solve that stange ~5px margin, i think things would look cleaner.

Reply:
Could you make a mockup in figma ( just edit the current one in molgenis ) to show what you suggest ? , i agree we need to clean this , but not yet sure on best layout , suggest to figure out design in figma and not in PR's or else it will take forever.